content/en/docs/configuration/acme/http01/_index.md

```diff
@@ -155,3 +168,163 @@ The added labels and annotations will merge on top of the cert-manager defaults,
 overriding entries with the same key.
 
 No other fields of the ingress can be edited.
+
+## Configuring the HTTP-01 Gateway API solver
+
+**FEATURE STATE**: cert-manager 1.5 [alpha]
+
+The Gateway and HTTPRoute resources are part of the [Gateway API][gwapi], a set
+of CRDs that you install on your Kubernetes cluster and which provide various
+improvements over the Ingress API.
+
+[gwapi]: https://gateway-api.sigs.k8s.io
+
+{{% pageinfo color="info" %}}
+
+📌  This feature requires to pass a feature flag to the cert-manager controller
+and the installation of the Gateway API CRDs.
```

Comment:
```suggestion
📌  This feature requires to passing a feature flag to the cert-manager controller
and the installation of the Gateway API CRDs.
```

Reply:
We should swap installing the CRDs and feature flag in this sentence since that is the order of the walk through